### PR TITLE
Improve flow for prerequisites

### PR DESCRIPTION
**Pull request by JeanPaulSG**
Improved the workflow around prerequisites and resource estimator. Brings this page in line with changes to other cloud specific docker compose guides. Removed language that is not accessible. 

issue: https://github.com/sourcegraph/product-education/issues/32 

## Test plan

Reviewers for technical assessment. 
PE team to proof read please. @blvckcoffee or @dulanism

### CODE REVIEW

File: doc/admin/deploy/docker-compose/digitalocean.md [hunk 1]
```diff
@@ -1,10 +1,24 @@
...
 
-This tutorial shows you how to deploy Sourcegraph via [Docker Compose](https://docs.docker.com/compose/) to a single Droplet running on DigitalOcean.
+This guide shows you how to deploy Sourcegraph via [Docker Compose](https://docs.docker.com/compose/) to a single Droplet running on DigitalOcean.
 
 > NOTE: Trying to decide how to deploy Sourcegraph? See [our recommendations](../index.md) for how to choose a deployment type that suits your needs.
 
 ---
+## Determine server and service requirements 
+
+Use the [resource estimator](../resource_estimator.md) to determine the resource requirements for your environment. You will use this information to set up the instance and configure the docker-compose. yaml file.
```
Comment (blvckcoffee) [inline doc/admin/deploy/docker-compose/digitalocean.md:10]:
```suggestion
Use the [resource estimator](../resource_estimator.md) to determine the resource requirements for your environment. You will use this information to set up the instance and configure the docker-compose YAML file. 
```

File: doc/admin/deploy/docker-compose/digitalocean.md [hunk 2]
```diff
@@ -1,10 +1,24 @@
...
 
 ---
+## Determine server and service requirements 
+
+Use the [resource estimator](../resource_estimator.md) to determine the resource requirements for your environment. You will use this information to set up the instance and configure the docker-compose. yaml file. 
+## Prepare a clone 
+
+To configure your Sourcegraph instance, you must create and use a fork of the reference repository. We strongly recommend that you create and run Sourcegraph from your own fork of the reference repository to track your customizations to the Sourcegraph Docker Compose YAML.
+
+To configure your Sourcegraph instance, you must create and use a fork of the reference repository - refer to the following steps, which use GitHub as an example, then return to this page:
```
[the remark this answers is not in the document]
Comment (JeanPaulSG) [reply doc/admin/deploy/docker-compose/digitalocean.md]: How did I miss that!? Thanks Joe.

File: doc/admin/deploy/docker-compose/digitalocean.md [hunk 3]
```diff
@@ -1,10 +1,22 @@
...
-This tutorial shows you how to deploy Sourcegraph via [Docker Compose](https://docs.docker.com/compose/) to a single Droplet running on DigitalOcean.
+This guide shows you how to deploy Sourcegraph via [Docker Compose](https://docs.docker.com/compose/) to a single Droplet running on DigitalOcean.
 
 > NOTE: Trying to decide how to deploy Sourcegraph? See [our recommendations](../index.md) for how to choose a deployment type that suits your needs.
 
 ---
+## Determine server and service requirements 
+
+Use the [resource estimator](../resource_estimator.md) to determine the resource requirements for your environment. You will use this information to set up the instance and configure the docker-compose YAML file. 
+## Prepare a clone
```
Comment (caugustus-sourcegraph) [inline doc/admin/deploy/docker-compose/digitalocean.md:11]: Should this be more descriptive, like "Prepare deployment configuration"? The clone phrasing all by itself seems odd without context.

**Review by caugustus-sourcegraph — APPROVED** [review]

File: doc/admin/deploy/docker-compose/digitalocean.md (hunk 3 above)
Comment (JeanPaulSG) [reply doc/admin/deploy/docker-compose/digitalocean.md:11]: Cool - Ill add some context after the Prepare a clone heading. (and rework the existing first sentence)

**Pull request merged**